app/helpers/application_helper/toolbar/miq_request_center.rb

```diff
@@ -27,7 +27,8 @@ class ApplicationHelper::Toolbar::MiqRequestCenter < ApplicationHelper::Toolbar:
       'fa fa-repeat fa-lg',
       N_('Reload the current display'),
       N_('Reload'),
-      :url_parms => "&display=miq_provisions"),
+      :url_parms => "&display=miq_provisions",
+      :klass     => ApplicationHelper::Button::MiqRequestReload),
```

Comment:
@vecerek I'm not able to display the `Reload` button on the detail of a request now. This should work, it's working on `master`

![screenshot from 2017-02-22 17-45-25](https://cloud.githubusercontent.com/assets/1187051/23222176/65972b92-f927-11e6-9eae-35091ca3a5ad.png)